build_projects/dotnet-cli-build/GetVersionBadgeMoniker.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.DotNet.Cli.Build
+{
+    public class GetVersionBadgeMoniker : Task
+    {
+        [Output]
+        public string VersionBadgeMoniker { get; set; }
+
+        public override bool Execute()
+        {
+            VersionBadgeMoniker = Monikers.GetBadgeMoniker();
```

Comment:
I don't think we need a task for this.  I think the logic in GetBadgeMoniker can be moved to MSBuild.